src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -1,19 +1,26 @@
 import React, { useCallback, useState, useMemo } from 'react';
```

Comment:
Couple of points regarding token management dialog:

1. When there is an error in a new address (in the input) the color doesn't change to red. I think to be consistent with other dialogs/forms it should change color.

2. When deleting token there are 2 warning messages for Appolo cache (i saw it only when deleting):

```
warning 1:
Cache data may be lost when replacing the tokenAddresses field of a ProcessedColony object.

To address this problem (which is not a bug in Apollo Client), define a custom merge function for the ProcessedColony.tokenAddresses field, so InMemoryCache can safely merge these objects:

  existing: ["0x8972e86549bb8E350673e0562fba9a4889d01637","0x7F176F534C31294fDbA77D80bB9d01279820C063"]
  incoming: ["0x7F176F534C31294fDbA77D80bB9d01279820C063"]

For more information about these options, please refer to the documentation:

  * Ensuring entity objects have IDs: https://go.apollo.dev/c/generating-unique-identifiers
  * Defining custom merge functions: https://go.apollo.dev/c/merging-non-normalized-objects

  
  warning 2:
  Cache data may be lost when replacing the tokens field of a ProcessedColony object.

To address this problem (which is not a bug in Apollo Client), define a custom merge function for the ProcessedColony.tokens field, so InMemoryCache can safely merge these objects:

  existing: [{"__ref":"Token:0x0000000000000000000000000000000000000000"},{"__ref":"Token:0x8972e86549bb8E350673e0562fba9a4889d01637"},{"__ref":"Token:0x7F176F534C31294fDbA77D80bB9d01279820C063"}]
  incoming: [{"__ref":"Token:0x0000000000000000000000000000000000000000"},{"__ref":"Token:0x7F176F534C31294fDbA77D80bB9d01279820C063"}]

For more information about these options, please refer to the documentation
...
```
 